pcsx2/GS/GSState.cpp

```diff
@@ -3361,46 +3369,14 @@ __forceinline void GSState::VertexKick(u32 skip)
 			__assume(0);
 	}
 
-	{
-		const GSVector4i voffset(GSVector4i::loadl(&m_context->XYOFFSET));
-
-		auto get_vertex = [&](u32 i) {
-			GSVector4i v(GSVector4i::loadl(&m_vertex.buff[m_index.buff[(m_index.tail - n) + (i)]].XYZ));
-			v = v.upl16(); // 16->32
-			v = v.sub32(voffset); // -= (OFX, OFY)
-			v = v.sra32(4); // >> 4
-			return v;
-		};
-
-		const GSVector4i xy0(get_vertex(0));
-		GSVector4i min, max;
-		if (m_vertex.tail == n)
-		{
-			min = xy0;
-			max = xy0;
-		}
-		else
-		{
-			min = temp_draw_rect.min_i32(xy0);
-			max = temp_draw_rect.zwzw().max_i32(xy0);
-		}
-
-		if constexpr (n > 1)
-		{
-			const GSVector4i xy1(get_vertex(1));
-			min = min.min_i32(xy1);
-			max = max.max_i32(xy1);
-
-			if constexpr (n > 2)
-			{
-				const GSVector4i xy2(get_vertex(2));
-				min = min.min_i32(xy2);
-				max = max.max_i32(xy2);
-			}
-		}
-
-		temp_draw_rect = min.upl64(max).rintersect(m_context->scissor.in);
-	}
+	// Update rectangle for the current draw. We can use the re-integer coordinates from min/max here.
+	const GSVector4i draw_min = pmin.zwzw();
+	const GSVector4i draw_max = pmax;
```

Comment:
Not used.

Reply:
I actually wrote up a comment further to this, but apparently I didn't submit it.  I was further asking if you meant to use that and did you need a swizzle like with draw_min?

Reply:
Apparently I just referenced pmax below instead.